### DISCUSSION TITLE
Export Datasets via CSV

### DISCUSSION THREAD

**Discussion by uladkaminski** [langfuse/langfuse#3438] [Ideas]
### Describe the feature or potential improvement

Traces and Session have a feature to export data with CSV and JSON formats using `BatchExportTableButton` component.

The idea is to add the similar functionality to allow export Dataset Items using web client.

**Comment by alexrosen** [comment]
@marcklingen - I believe you could mark this one as implemented now.

**Reply by marcklingen** [reply]
this is not available yet as datasets can only be exported via the sdk/api right now, tracking exports via UI here: https://github.com/langfuse/langfuse/issues/6090

**Comment by marcklingen** [comment]
This is available now: https://langfuse.com/changelog/2025-06-23-exports-datasets-audit-logs
